products/samsungmobile.md

```diff
@@ -107,7 +332,7 @@ releases:
     releaseDate: 2021-05-25
 
 -   releaseCycle: "Galaxy A82 5G"
```

Comment:
Note: This one has two name : https://www.gsmarena.com/samsung_accidentally_confirms_the_galaxy_a82_5g_existence-news-48968.php.